Makefile

```diff
@@ -1,8 +1,7 @@
 #   Makefile for T3
 
 install:
-	bash devtools/install_arc.sh
```

Comment:
are we removing these two scripts?

Reply:
Not right now. I may fix them up for users to use them separately if they prefer. But for the mean time I will leave them untouched and makes changes in another PR